build_variables.bzl

```diff
@@ -920,6 +921,7 @@ libtorch_python_core_sources = [
     "torch/csrc/dynamo/guards.cpp",
     "torch/csrc/dynamo/init.cpp",
     "torch/csrc/functorch/init.cpp",
+    "torch/csrc/mps/Module.cpp",
```

Comment:
All the backend modules are defined there (cuda, dynamo, functorch, etc.). The new file Module.cpp needs to be built otherwise the MPS Module interface will not be recognized by pytorch.